django/contrib/admin/options.py

```diff
@@ -1183,6 +1194,21 @@ def response_change(self, request, obj):
         Determines the HttpResponse for the change_view stage.
         """
 
+        if IS_POPUP_VAR in request.POST:
+            to_field = request.POST.get(TO_FIELD_VAR)
+            if to_field:
+                attr = str(to_field)
+            else:
+                attr = obj._meta.pk.attname
+            value = escape(request.resolver_match.args[0])
```

Comment:
a comment about what `request.resolver_match.args[0]` is would be helpful as it's not obvious